public/styles/index.css

```diff
@@ -209,6 +209,25 @@
   display: none;
 }
 
+.tos-section {
+  background: linear-gradient(var(--human-pink-light), white);
+  color: var(--human-blue);
+  padding: 15% 10% 5% 10%;
+}
+
+.tos-section h3 {
+  padding: 1rem 0;
+}
+.privacy-section {
+  background: linear-gradient(var(--human-pink-light), white);
+  color: var(--human-blue);
+  padding: 15% 10% 5% 10%;
+}
+
+.privacy-section h3 {
+  padding: 1rem 0;
+}
```

Comment:
A few comments:
- Please merge this branch with the `main` branch and extract this code to a new SCSS file.
- The properties for `tos` and `privacy` are identical, so we can use the same selectors for both HTML files.
- The new SCSS file should use the `$human-xxx` format for the colors instead of the `var(--human-xxx)` we have now.
- The padding with `%` is a bit problematic for small screens. The text hides the Roar logo.

Reply:
merged with main branch. 
combined selector. 
changed color variable
change top padding to rem, text under Roar logo on small screens.